packages/odyssey-storybook/src/components/odyssey-mui/MenuButton/MenuButton.stories.tsx

```diff
@@ -252,6 +257,14 @@ export const ButtonVariant: StoryObj<MenuButtonProps> = {
       <MenuItem key="2">Edit configuration</MenuItem>,
       <MenuItem key="3">Launch</MenuItem>,
     ],
+    id: "floating",
+  },
+  play: async ({ canvasElement, step }: PlaywrightProps<MenuButtonProps>) => {
+    await step("Filter and Select from listbox", async () => {
+      const canvas = within(canvasElement);
+      const button = canvas.getByRole("button");
+      expect(button).toHaveAttribute("id", "floating-button");
+    });
```

Comment:
I think we can be more specific and select a button with a specific label; otherwise when reading this test, I don't know which button you're selecting. At first, I thought it was a menu item, but now I can see it's the menu opener button.

```suggestion
    await step("Filter and Select from listbox", async () => {
      const canvas = within(canvasElement);
      const button = canvas.getByRole("button", { name: "More actions" });
      expect(button).toHaveAttribute("id", "floating-button");
    });
```

The benefit of utilizing more-specific selectors from Testing Library is we get free testing from it too. Now we're verifying we can not only see a button, but that the ARIA label text matches what we'd expect a screen reader to see. It kills 2 birds with one stone.